pokecli.py

```diff
@@ -116,6 +116,7 @@ def get_commit_hash():
 
                 while True:
                     bot.tick()
+                    inventory.refresh_inventory()
```

Comment:
Should be in the heartbeat function, not everyone uses the webui and not everyone should have to suffer a potential ban by doing api calls every tick because of a small minority who want a fancy gui